projects/plugins/jetpack/_inc/client/recommendations/prompts/resource-prompt/index.jsx

```diff
@@ -155,14 +157,20 @@ const ResourcePromptComponent = props => {
 				<div className="jp-recommendations-question__install-section">
 					{ ! hasNoAction ? (
 						<>
-							<ExternalLink
-								type="button"
-								className="dops-button is-rna is-primary"
-								href={ ctaLink }
-								onClick={ onResourceLinkClick }
-							>
-								{ ctaText }
-							</ExternalLink>
+							{ ctaLinkIsExternal ? (
+								<ExternalLink
+									type="button"
+									className="dops-button is-rna is-primary"
+									href={ ctaLink }
+									onClick={ onResourceLinkClick }
+								>
+									{ ctaText }
+								</ExternalLink>
+							) : (
+								<Button rna href={ ctaLink } onClick={ onResourceLinkClick }>
+									{ ctaText }
+								</Button>
+							) }
```

Comment:
This logic is same as the one used [here](https://github.com/Automattic/jetpack/blob/4f93290da69ab96b6e0a57354c2d82354817eefe/projects/plugins/jetpack/_inc/client/recommendations/prompts/feature-prompt/index.jsx#L202-L215)